docs/source/network/network.md

```diff
@@ -4,7 +4,7 @@ This topic will describe, **at a conceptual level**, how Hyperledger Fabric allo
 
 After reading this topic and understanding the concept of policies, you will have a solid understanding of the decisions that organizations need to make to establish the policies that control a deployed Hyperledger Fabric network. You'll also understand how organizations manage network evolution using declarative policies -- a key feature of Hyperledger Fabric. In a nutshell, you'll understand the major technical components of Hyperledger Fabric and the decisions organizations need to make about them.
 
-Note: in this topic, we'll refer to the structure of a network that does not have a "system channel", a channel run by the ordering service that ordering nodes are bootstrapped with. For a version of this topic that does use the system channel, check out [Blockchain network](https://hyperledger-fabric.readthedocs.io/en/release-2.2/network/network.html).
+Note: in this topic, we'll refer to the structure of a network that does not have a "system channel" since it is no longer supported from release v3.0, a channel run by the ordering service that ordering nodes are bootstrapped with.
```

Comment:
drop "a channel run by the ordering service that ordering nodes are bootstrapped with."

Reply:
Again, I don't think a Note is needed here at all, since there is no other option now.